### DISCUSSION TITLE
如何在VcMeasurements量算工具中获取到量算的结果及area外的其他量算结果的label可不可以展示自定义内容

### DISCUSSION THREAD

**Discussion by skyBlueRain** [zouyaoji/vue-cesium#645] [Q&A]
在量算结束后改该如何获取量算结果喃？area量算外的其他量算工具有没有类似的areaFormaater的方式来展示自定义类容或者单位换算喃？
![1735093967625](https://github.com/user-attachments/assets/f630acc4-c1dd-4b17-b946-785f8dcde7ab)

**Comment by zouyaoji** [comment]
https://github.com/zouyaoji/vue-cesium/discussions/608
